pkgs/applications/video/manim/default.nix

```diff
@@ -69,14 +69,14 @@ let
 in python.pkgs.buildPythonApplication rec {
   pname = "manim";
   format = "pyproject";
-  version = "0.16.0.post0";
+  version = "0.17.3";
   disabled = python3.pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner  = "ManimCommunity";
-    repo = pname;
+    repo = "manim";
```

Comment:
This change is not needed. It's common practice to use `pname`.

Reply:
Actually I have originally made this change because @SuperSandro2000 told me several time to not use `repo = pname`.
I will remove it then I guess.

Reply:
That is still correct but it is probably not required to change this in existing packages.